share/normalize/links.py

```diff
@@ -832,5 +892,52 @@ def execute(self, obj):
                 break
 
         if not final[0]:
-            raise ValueError('\'{}\' could not be identified as an Identifier.'.format(obj))
```

Comment:
When would you want to not fail here? As Identifiers are fairly important, I'd rather not make them implicitly fire and forget.